Version.d.tpl

```diff
@@ -113,16 +101,8 @@ private void buildPredefinedVersions()
 
 *******************************************************************************/
 
-private template Id (_tpl_istring name)
+private template Id (string name)
 {
-    version (D_Version2)
-    {
-        mixin("static immutable Id = `version (` ~ name ~ `) version_info[\"ver_`
-             ~ name ~ `\"] = \"` ~ name ~ `\";`;");
-    }
-    else
-    {
-        const _tpl_istring Id = `version (` ~ name ~ `) version_info["ver_` ~ name ~
-            `"] = "` ~ name ~ `";`;
-    }
+    static immutable Id = `version (` ~ name ~ `) version_info[\"ver_`
+         ~ name ~ `\"] = \"` ~ name ~ `\";`;
```

Comment:
I think the `\` should have been removed from this string -- they were needed because of the surrounding `mixin("...")` but now they insert actual backslashes.